README.md

```diff
@@ -38,7 +38,7 @@ This example opens a serial port and echoes back everything that is read.
 ```rust
 use serial2::SerialPort;
 
-// On Windows, use something like "COM1".
+// On Windows, use r"\\\\.\\COM1" or just "COM1".
```

Comment:
```suggestion
// On Windows, use something like "COM1".
// For COM ports above COM9, you need to use the win32 device namespace, for example "\\.\COM10" (or "\\\\.\\COM10" with string escaping).
// For more details, see: https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file?redirectedfrom=MSDN#win32-device-namespaces
```

Reply:
Thanks for the PR. I like to idea of explaining the win32 device namespace, but I would make it a bit more explicit with regards to escaping. And I'd like to add a link to the microsoft documentation.